.codegen/api.go.tmpl

```diff
@@ -159,6 +159,79 @@ func (a *{{.Service.Name}}API) {{.PascalName}}AndWait(ctx context.Context{{if .R
 	return wait.Get()
 }
 {{end}}{{if .Pagination}}
+{{.Comment "// " 80}}
+func (a *{{.Service.Name}}API) {{.PascalName}}Iter(ctx context.Context{{if .Request}}, request {{.Request.PascalName}}{{end}}) func () (*{{.Pagination.Entity.PascalName}}, error) {
+	var err error
+	finished := false
+	var results []{{.Pagination.Entity.PascalName}}
+	{{ if .Pagination.MultiRequest -}}
+	ctx = useragent.InContext(ctx, "sdk-feature", "pagination")
+	{{- end }}
```

Comment:
This user agent addition is no longer relevant now that the default behavior is to list everything.